lib/player.js

```diff
@@ -878,8 +878,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
 
     // We do not support iOS 9, 10, 11 or 12, nor those same versions of
     // desktop Safari.
+    // Only check for Safari version if the platform is not Amazon Fire TV
+    // because Fire TV devices have a `Version/4.0` in their user agents
+    // even though they run on a modern chrome version
     const safariVersion = shaka.util.Platform.safariVersion();
-    if (safariVersion && safariVersion < 13) {
+    if (!shaka.util.Platform.isAmazonFireTV() &&
```

Comment:
I think it might be best to move this check into isApple(), which is used inside safariVersion() to gate version parsing.  safariVersion() should return null if it's not Safari, so excluding fire TV from isApple() is a better fix.

Reply:
Thanks, Joey. I've moved the check into `isApple()` function.